### PR TITLE
Latex formula transformation

### LINKED ISSUE #23: Transform latex parsed syntax tree to Plurimath Formula
Transform latex parsed syntax tree to Plurimath Formula object.

### PR DESCRIPTION
**Pull request by suleman-uzair**
Added transformation rules for the ```syntax tree```.

closes #23

### CODE REVIEW

File: lib/plurimath/latex/parser.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+      def initialize(text)
+        @text = text.gsub(" ", "").gsub("\n", "")
+      end
+
+      def parse
+        tree_t = Plurimath::Latex::Parse.new.parse(text)
+        formula = Plurimath::Latex::Transform.new.apply(tree_t)
+        formula = [formula] unless formula.is_a?(Array) || formula.nil?
+        return if formula.nil?
```
Comment (HassanAkbar) [inline lib/plurimath/latex/parser.rb:19]: I think `nil` should not be returned, always return the formula object even if it is empty.

File: spec/plurimath/math/function/table_spec.rb [hunk 2]
```diff
@@ -3,25 +3,32 @@
 RSpec.describe Plurimath::Math::Function::Table do
 
   describe ".initialize" do
-    subject(:table) { Plurimath::Math::Function::Table.new(first) }
+    subject(:table) { Plurimath::Math::Function::Table.new([first], second, third) }
```
Comment (HassanAkbar) [inline spec/plurimath/math/function/table_spec.rb:6]: Can you use a real example here?

**Review by HassanAkbar — APPROVED** [review]
LGTM.

**Comment by ronaldtse** [comment]
Let's get this merged! Thanks @suleman-uzair !

**Pull request merged**